packages/core/src/reducers/suggestedActions.js

```diff
@@ -6,7 +6,11 @@ const DEFAULT_STATE = [];
 export default function suggestedActions(state = DEFAULT_STATE, { payload = {}, type }) {
   switch (type) {
     case SET_SUGGESTED_ACTIONS:
-      state = [].slice.call(payload.suggestedActions || []);
+      if ((payload.suggestedActions || []).length) {
+        state = [].slice.call(payload.suggestedActions || []);
+      } else {
+        state = DEFAULT_STATE;
```

Comment:
Why is this necessary? It should already be in it's default state since it's reset before every incoming activity.

Reply:
Accessibility issue, it was rerendering every time even though technically the content isn't changed.